{{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py

```diff
@@ -1,7 +1,7 @@
 from anyblok_pyramid.tests.testcase import PyramidBlokTestCase
 
 
-class TestCanigooRestApi(PyramidBlokTestCase):
```

Comment:
I am wondering, who write the example :D

Reply:
Me too ;-)